evol/step.py

```diff
@@ -60,9 +60,17 @@ def apply(self, population) -> Population:
 
 
 class LogStep(EvolutionStep):
+    def __init__(self, name, every=1, **kwargs):
+        EvolutionStep.__init__(self, name, **kwargs)
+        self.count = 0
+        self.every = every
 
     def apply(self, population) -> Population:
-        return population.log(**self.kwargs)
+        self.count += 1
```

Comment:
Since this functionality mirrors that of the `CheckpointStep`, should we deduplicate that by making an intermediate `EvolutionEveryStep`?

Reply:
if we have a 3rd method that does this, sure. otherwise it feels a bit premature to me.